.changeset/free-words-design.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/factory': patch
+---
+
+Improved Factory reviews by checking linked issue scope, requiring approved feature issues, independently establishing behavior-changing claims, requiring integration-level evidence for model-provider behavior, and comparing new pluggable implementations with analogous ones.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)